swift/ios-shoppe-demo/Views/ProductSummaryDetailCell.swift

```diff
@@ -14,4 +14,16 @@ class ProductSummaryDetailCell: UITableViewCell {
     @IBOutlet weak var shippingLabel: UILabel!
     @IBOutlet weak var nameLabel: UILabel!
     @IBOutlet weak var shippingAddress: UILabel!
+
+    // TODO: - Add masking after FSManager is Approved. var fsManager = FullStoryManager.shared 
```

Comment:
Let's remove WIP from master.

Reply:
This can go into the FSManager PR, and you can introduce the Manager with dependent code.